app/AppKernel.php

```diff
@@ -205,7 +205,8 @@ public function registerBundles(): array
             $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
             $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
             $bundles[] = new Webfactory\Bundle\ExceptionsBundle\WebfactoryExceptionsBundle();
-            $bundles[] = new Fidry\PsyshBundle\PsyshBundle();
+            // Removed temporarily for https://github.com/mautic/mautic/pull/9602, can be re-added in Mautic 4+
+            // $bundles[] = new Fidry\PsyshBundle\PsyshBundle();
```

Comment:
Newer versions of the PsyshBundle have Symfony 4+ as a hard dependency. Will fix this in the Symfony 4 PR as soon as this one is merged https://github.com/mautic/mautic/pull/9409